randovania/games/am2r/logic_database/Hydro Station.txt

```diff
@@ -612,9 +612,12 @@ Extra - minimap_data: [{'x': 33, 'y': 23}, {'x': 34, 'y': 23}, {'x': 35, 'y': 23
                   Gravity Suit and Morph Ball and Speed Booster and Shinesparking Tricks (Beginner)
                   Any of the following:
                       Power Grip or Can Use Spring Ball
+                      # https://www.youtube.com/watch?v=mQahK4kIL3w
```

Comment:
can you add another comment to the yt vid?

Reply:
What comment?

Reply:
something like "mid air shinespark: [yt link]"